skimage/filters/thresholding.py

```diff
@@ -66,9 +71,13 @@ def _try_all(image, methods=None, figsize=None, num_cols=2, verbose=True):
 
     i = 1
     for name, func in methods.items():
+        # Use precomputed histogram for supporting functions.
```

Comment:
```suggestion
        # Use precomputed histogram for supporting functions
```
just because other similar comments don't end with a period